sections/docker/smaller_base_images.md

```diff
@@ -0,0 +1,13 @@
+# Prefer smaller Docker base images
+
+Large Docker images can lead to higher exposure to vulnerabilities and increased resource consumption. Often, you don't need certain packages installed at runtime that are needed for building.
+Pulling and storing larger images will become more expensive at scale, when dealing with larger images. Minimal images may not come with common libraries needed for building native modules or packages
+useful for debugging (e.g. curl) pre-installed, by design. Using the Alpine Linux variants of images, can lead to a reduced footprint in terms of resources used and the amount of attack vectors
+present in fully-featured systems. The Node.js v14.4.0 Docker image is ~345MB in size versus ~39MB for the Alpine version.
```

Comment:
Maybe add a statement like "That's almost a 10x reduction" for more impact?

Reply:
@BrunoScheufler Consider also including 'Slim' which is very popular. Maybe make the point that working with ~500MB images will make it feasible to push changes only from high-bandwidth lines (happened to me a lot that I couldn't push in planes or trains)